pkg/beholder/config.go

```diff
@@ -0,0 +1,58 @@
+package beholder
+
+import (
+	"time"
+
+	otelattr "go.opentelemetry.io/otel/attribute"
+)
+
+type Config struct {
+	InsecureConnection       bool
+	CACertFile               string
+	OtelExporterGRPCEndpoint string
+
+	PackageName string
```

Comment:
@krebernisak 
> How is PackageName expected to be used? Can we document it here?

The package name is a high-level name for an app or a sub-module that is instrumented.
Is intended to be used for routing in the Beholder/OpenTelemetry pipeline. 
Once that telemetry data is persisted in respective sink/storage it will be used for querying/filtering I would imagine.

> if this attribute will need to change often?

No, it should be fairly static per app/sub-module

Reply:
I can certainly add support for `client.ForPackage(packageName)` builder

Reply:
@krebernisak added `client.ForPackage(packageName)` [here](https://github.com/smartcontractkit/chainlink-common/pull/694/commits/b69b21e9e30864a913aed32713de6ec0d4255d4e)